platforms/corda-ent/configuration/roles/setup/node/tasks/main.yaml

```diff
@@ -0,0 +1,101 @@
+#This task creates the value file for the node registration as well the db required and push them to the git repository
+
+# Create ambassador certificates for node
+- name: Create ambassador certificates for node
+  include_role:
+    name: create/certificates/ambassador/node
+
+# Create value file for node registration
+- name: Create value file for node registration
+  include_role:
+    name: setup/node_registration
+
+# Wait for node registration job to complete
+- name: "Waiting for node registration job to complete"
+  include_role:
+    name: "{{ playbook_dir }}/../../shared/configuration/roles/check/helm_component"
+  vars:
+    component_type: "Job"
+    namespace: "{{ org.name | lower }}-ent"
+    component_name: "{{ peer.name }}registration-registration"
+    kubernetes: "{{ org.k8s }}"
+  loop: "{{ org.services.peers }}"
+  loop_control:
+    loop_var: peer
+
+- name: "Create value file for node"
+  include_role:
+    name: helm_component
+  vars:
+    git_url: "{{ org.gitops.git_ssh }}"
```

Comment:
Every value does not need to be a variable here. All fixed values should be hardcoded in the tpl file. Unnecessary variable creation in Ansible should be avoided.

Reply:
refactored